code-pushup/ng-demo
# migrate to latest Nx and Angular 19

1. Run `npx nx migrate latest`
2. make sure `npm i` passes with latest dependencies
3. Run `npx nx migrate latest --runMigrations`
